livechat/reports/client.py

```diff
@@ -421,6 +421,64 @@ def engagement(self,
                                  json=payload,
                                  headers=headers)
 
+    def greetings_conversion(self,
+                             distribution: str = 'day',
+                             timezone: str = None,
+                             filters: dict = None,
+                             payload: dict = None,
+                             headers: dict = None) -> httpx.Response:
```

Comment:
Why `distribution: str = 'day'` I mean it is by default day if we pass this or not. So I think this is redundant 🤔

Reply:
Applies to others